lib/tasks/onetime.rake

```diff
@@ -717,4 +717,24 @@ namespace :onetime do
       child_id: child_id
     )
   end
+
+  # Services should always have a corresponding schedule objects, even if they inherit from resource
+  # This task backfills missing schedule objects for some services
+  desc 'Add schedule objects to services that are missing them'
+  task add_missing_schedule_objects: :environment do
+    Schedule.transaction do
+      # Get services with no schedule objects
+      rogue_services = Service.joins('LEFT JOIN schedules '\
+          'ON services.id = schedules.service_id '\
+          'WHERE schedules.id is NULL')
```

Comment:
This is fine, especially since this is just a one-time task, but for future reference, you can do multiline strings in Ruby with the special `%` strings: https://stackoverflow.com/a/18478045, https://en.wikibooks.org/wiki/Ruby_Programming/Syntax/Literals#The_%_Notation

It'd look kind of like:

```rb
      rogue_services = Service.joins(%{
        LEFT JOIN schedules 
          ON services.id = schedules.service_id
          WHERE schedules.id is NULL
      %})
```

Note that this will insert actual newline characters and excessive spaces in the string due to the indentation, but it doesn't matter for SQL statements, since SQL doesn't care about having the extra whitespace. Also for future reference, if you did care about excessive spaces, you could also use the Ruby 2.3.0 syntax for squiggly heredocs: https://en.wikibooks.org/wiki/Ruby_Programming/Syntax/Literals#The_%_Notation

Reply:
oh great! thank you 😃